internal/cli/options.go

```diff
@@ -63,6 +64,7 @@ func (o *Options) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&o.validatePath, "validate", "", "Path to a file to perform a validation operation on.")
 	fs.BoolVar(&o.merge, "merge", false, "Perform a merge operation between --lhs and --rhs")
 	fs.BoolVar(&o.compare, "compare", false, "Perform a compare operation between --lhs and --rhs")
+	fs.StringVar(&o.fieldset, "fieldset", "", "Build a field set for the given file.")
```

Comment:
`"Path to a file for which we should build a fieldset"`

(otherwise people will have no idea how to pass "the given file")